cpp/src/gandiva/gdv_function_stubs_test.cc

```diff
@@ -766,4 +766,64 @@ TEST(TestGdvFnStubs, TestCastVarbinaryFloat8) {
   ctx.Reset();
 }
 
+TEST(TestGdvFnStubs, TestMaskFirstN) {
+  gandiva::ExecutionContext ctx;
+  int64_t ctx_ptr = reinterpret_cast<int64_t>(&ctx);
+  int32_t out_len = 0;
+
+  const char* result = gdv_fn_mask_first_n(ctx_ptr, "0123456789", 10, 10, &out_len);
```

Comment:
Please add a test where input length < number of chars to mask

Reply:
Done